scripts/hub_deploy.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash -e
+
+script_dir=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
+base_dir=$(cd "${script_dir}/.." && pwd)
+
+build_dir="${base_dir}/build/appsody_stacks"
+
+
+prereqs() {
+    docker_cmd="docker"
+    command -v $docker_cmd >/dev/null 2>&1 || { docker_cmd="podman"; }
+    command -v $docker_cmd >/dev/null 2>&1 || { echo "Unable to deploy images: docker or podman are not installed."; exit 1; }
+
+    command -v oc >/dev/null 2>&1 || { echo "Unable to mirror images or deploy stack-hub-index: oc is not installed."; exit 1; }
+}
+
+image_push() {
+    local name=$@
+    echo "== Pushing $name"
+    if [ "$docker_cmd" == "podman" ]; then
+        podman push --tls-verify=false $name
+    else
+        docker push $name
+    fi
+}
+
+image_mirror() {
+    local file=$1
+    oc image mirror -f "$file" --insecure
+}
+
+get_route() {
+    for i in 1 2 3 4 5 6 7 8 9 10; do
```

Comment:
What is the purpose of this for loop?

Reply:
It's a silly look to wait a few seconds. Sometimes it takes a bit for the route information (the endpoint) to be populated.